src/com/tigersale/userInterface/customerUser/CustomerUserHomeView.java

```diff
@@ -31,7 +32,7 @@ public CustomerUserHomeView(Scanner scanner, CustomerUser user)
     /**
      * This method represents the logged in users home view
      */
-    public void runCustomerUserHomeView()
+    public void runCustomerUserHomeView(CustomerUser user)
```

Comment:
This view contains the customerUser already as a private field of the class.  You shouldnt be passing it in.